doc/utilities.md

```diff
@@ -0,0 +1,24 @@
+# Invalid_activities rake task
+
+## Context
+
+`invalid_activities` is a rake task that produces a CSV file that contains
```

Comment:
This paragraph feels more like a commit message for me, not sure we need it?

Reply:
Okay. I wasn't sure how much of an explanation of this rake task to give to future devs. I agree that is repetitive, considering that the commit message of the task is quite similar. I'll remove it and merge the PR if that's okay?